src/modules/src/stabilizer.c

```diff
@@ -184,7 +184,7 @@ void stabilizerInit(StateEstimatorType estimator)
 
   sensorsInit();
   stateEstimatorInit(estimator);
-  controllerInit(ControllerTypeAny);
```

Comment:
Could you please return this to ControllerTypeAny? The controller type will be set later in the stabilizer.c code

Reply:
Done :)

Reply:
thanks! more comments are now coming up